apps/admin-gui/src/app/shared/components/member-overview-membership/member-overview-membership.component.ts

```diff
@@ -78,6 +78,9 @@ export class MemberOverviewMembershipComponent implements OnChanges {
     const dialogRef = this.dialog.open(ChangeVoExpirationDialogComponent, config);
     dialogRef.afterClosed().subscribe(success => {
       if (success) {
+        if(success !== true){
+          this.member = success;
+        }
```

Comment:
Honestly this is not a good code practice. My suggestion is to return tuple (success, meber) from the dialog.